k0765065/quiz1.py

```diff
@@ -0,0 +1,39 @@
+############################
+#
+# Please complete this quiz
+#
+
+# declare an int variable named total with the value of 0
+z = 0
+
+# declare an array named numbers with all numbers 0 - 20
+number_list = []
+x = 0
+
+while x < 21:
```

Comment:
This works, but I have to read it to figure out what it does. `range(0,21)` tells me immediately